mesheryctl/internal/cli/root/system/model_test.go

```diff
@@ -173,9 +173,9 @@ func TestModelViewCmd(t *testing.T) {
 			r, w, _ := os.Pipe()
 			os.Stdout = w
 			_ = utils.SetupMeshkitLoggerTesting(t, false)
-			ModelCmd.SetArgs(tt.Args)
-			ModelCmd.SetOutput(rescueStdout)
-			err = ModelCmd.Execute()
+			SystemCmd.SetArgs(tt.Args)
+			SystemCmd.SetOutput(rescueStdout)
+			err = SystemCmd.Execute()
```

Comment:
Usage of `SystemCmd`? I see that `model` is referenced via args... But why not we are using `ModelCmd` here?

Reply:
Yes, `ModelCmd` is attached to the system command. Changing it to `ModelCmd` causes unit test failure. `ModelCmd` was previously commented out in the system.go, but now that it's linked to the system command, we need `SystemCmd`.

Reply:
Using the previous `ModelCmd` instead of `SystemCmd`, also `ModelCmd` is still attached to `exp` parent command.